plugins/plugin-client-common/src/components/Views/Terminal/Block/Output.tsx

```diff
@@ -203,7 +204,12 @@ export default class Output extends React.PureComponent<Props, State> {
           className={
             'repl-result' +
             (isOops(this.props.model) ? ' oops' : '') +
-            (isWithCompleteEvent(this.props.model) && isMixedResponse(this.props.model.response) ? ' flex-column' : '')
+            (isWithCompleteEvent(this.props.model) && isMixedResponse(this.props.model.response)
+              ? ' flex-column'
+              : '') +
+            (isWithCompleteEvent(this.props.model) && isRadioTable(this.props.model.response)
+              ? ' overflow-visible'
+              : '')
```

Comment:
Here I added the class `overflow-visible` to `repl-result` when the response is `RadioTable`.